### PR TITLE
Add documentation links

### PR DESCRIPTION
**Pull request by kamikazechaser**
Library users should easily find the documentation especially those viewing from npmjs.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # irc-framework
-A better IRC framework for node.js. For bots and full clients
+A better IRC framework for node.js. For bots and full clients. Read the [documentation](https://github.com/kamikazechaser/irc-framework/blob/master/docs/clientapi.md).
```
Comment (MaxLeiter) [inline README.md]: You linked to your fork

**Comment by kamikazechaser** [comment]
My bad! Will change asap!

**Comment by kamikazechaser** [comment]
Fixed.

**Comment by MaxLeiter** [comment]
Thanks!

**Pull request merged**